spec/acceptance/systemd_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'systemd' do
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<-PUPPET
+      include systemd
+      PUPPET
+    end
```

Comment:
Only partly related discussion and you probably know this, but the short version of this is:
```suggestion
    let(:manifest) { 'include systemd' }
```

I wonder if we should encourage one style over the other or just see what individuals prefer.

Reply:
There HERE document has the advantage that you can correct puppet style quoting once the line becomes more interesting.

Reply:
I agree and I think for consistency that's why I also commonly use heredoc. It makes it easy to add multiple lines without reformatting the whole block.